db/schema.rb

```diff
@@ -142,4 +142,22 @@
 
   add_index "versions", ["item_type", "item_id"], name: "index_versions_on_item_type_and_item_id", using: :btree
 
+  create_table "views", force: :cascade do |t|
```

Comment:
Oh I know, you must've run `rails generate devise views` instead of `rails generate devise:views`. Please remove these changes.